src/main/resources/issues_conditions.json

```diff
@@ -0,0 +1,7 @@
+###vms
+$.ManageIQ::Providers::Vmware::InfraManager[*].vms[?(@.host == null )].id
```

Comment:
I think it is worth adding a basic test for every possible value of `host`. I've created a PR to your branch so you can merge it or just take it as an example. https://github.com/m-brophy/xavier-payload-sanitizer/pull/1

- We could also add a very similar test to verify cpu_total_cores, ems_ref, and ems_cluster